modules/runner/src/integrationTest/java/org/apache/ignite/internal/ssl/ItSslTest.java

```diff
@@ -104,10 +104,18 @@ void clientCouldConnectWithoutSsl() throws Exception {
                 assertThat(client.clusterNodes(), hasSize(2));
             }
         }
+
+        @Test
+        @DisplayName("Jdbc driver could establish the connection when SSL disabled")
+        void jdbcCouldConnectWithoutSsl() throws SQLException {
```

Comment:
```suggestion
        void jdbcCanConnectWithoutSsl() throws SQLException {
```
Also,
Can secure jdbc connection be established with insecure server?
I guess we should expect an exception in that case, as client requirements looks more strict.
Let's add a test for this.